scripts/examples/gn_efr32_example.sh

```diff
@@ -137,6 +137,10 @@ else
                 optArgs+="enable_sleepy_device=true chip_openthread_ftd=false "
                 shift
                 ;;
+            chip_disable_wifi_ipv4)
```

Comment:
Please maintain the command arguments structure

For preset command use the two dashes `--chip_disable_wifi_ipv4` else you can remove this new section and just pass the whole variable with the value aka `chip_disable_wifi_ipv4=true` and it will be passed correctly to the gn args

Reply:
added two dashes for preset command